snapcraft/commands/upload.py

```diff
@@ -54,13 +54,13 @@ def main(argv=None):
 
     # make sure the full lifecycle is executed
     yaml_config = snapcraft.yaml.load_config()
-    snap_name = _format_snap_name(yaml_config.data)
+    snap_package = _format_snap_name(yaml_config.data)
```

Comment:
Don't call it package; its a language we are trying to get rid of, just call it `snap`.